cordova-plugin-medic/www/jasmine-jsreporter.js

```diff
@@ -1,214 +0,0 @@
-/*
-  This file is part of the Jasmine JSReporter project from Ivan De Marino.
-
-  Copyright (C) 2011 Ivan De Marino (aka detro, aka detronizator), http://blog.ivandemarino.me, ivan.de.marino@gmail.com
```

Comment:
Particularly this file seems dead code. I assumed Nikhil is asking about all deleted files in this PR. Sorry for confusion.